Dockerfile.moc-xdmod

```diff
@@ -89,6 +90,14 @@ RUN chmod g+rwX /usr/bin/xdmod-openstack-hypervisor \
     && chmod -R g+rwX /etc/httpd \
     && chgrp -R 0 /etc/httpd 
 
+# openshift reporting
+# checkout https://github.com/OCP-on-NERC/xdmod-openshift-scripts
+# copy openshift xdmod script to /usr/bin/xdmod-openshift.sh (to be created)
```

Comment:
I would go with /usr/bin/xdmod-openshift-reporting as that seems to fit with their naming convention (just considering all of the xdmod-... scripts that exist in /usr/bin.